blog.html

```diff
@@ -298,6 +299,22 @@ <h2 class="blog-title">Retriever-Aware Training (RAT): Are LLMs memorizing or un
                 </div>
             </a>
         </div>
+
+        <!-- Blog Post 4 -->
+        <div class="blog-post">
+            <a href="blogs/5_api_zoo.html">
+                <div class="blog-content">
+                    <h2 class="blog-title">The API Zoo: A Keystone for Building API-connected LLMs</h2>
+                    <div class="author-date">
+                        <p class="author"><a href="https://www.linkedin.com/in/ramanvarma/">Raman Varma</a></p>
+                        <p class="date">Feb 16, 2024</p>
+                    </div>
+                    <a href="blogs/5_api_zoo.html">
+                        <img src="assets/img/gorilla_method.png" alt="Blog Post 4 Image" style="width: 100%;">
```

Comment:
Maybe you use the screenshot of the API Zoo landing page for image rather than re-using the gorilla image?